docs/tutorials/vault/creating-vault-issuers.rst

```diff
@@ -67,7 +68,9 @@ The Vault appRole credentials are supplied as the
 Vault authentication method using the appRole created in Vault. The secretRef
 references the Kubernetes secret created previously. More specifically, the field
 *name* is the Kubernetes secret name and *key* is the name given as the
-key value that store the *secretId*.
+key value that store the *secretId*. The optional attribute *authPath* specifies
+where the AppRole authentication is mounted in Vault. The *authPath* default value
+is *approle*.
```

Comment:
`s/authPath/path` - also, is this comment still true?

Reply:
Yes if the path is empty it still default to approle.